src/routes/adminRoutes.js

```diff
@@ -13,4 +15,9 @@ router.use(
   })
 );
 
+
+router
+  .route('/admin/api/albums')
```

Comment:
Because we "mount" this router in `expressServer.js` to the `/admin` route: https://github.com/Code-the-Dream-School/dd-prac-team4-back/blob/PS97-bulk-edit-prices/src/expressServer.js#L122

That means that any routes we define in this file will be _appended_ to `<SERVER_URL>/admin`

So what you have here will be trying to handle calls from `/admin/admin/api/albums`.

But the call in the template is trying to call `/admin/api/album`. Express thinks that path doesn't exist and then returns a 404

```suggestion
  .route('/api/albums')
```

Reply:
Thank you very much